lib/scripts/init_repo.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e #-x #ONLY FOR DEV
+
```

Comment:
if [ -z "$4" ]; then
	echo "$0 repo_url [clone_from]"
        echo ...short description what this script do
	exit 1
fi